package.json

```diff
@@ -71,7 +71,7 @@
     "handlebars-helpers": "^0.10.0",
     "helper-markdown": "^1.0.0",
     "helper-md": "^0.2.2",
-    "jest-cli": "^22.1.4",
+    "jest-cli": "^21.2.1",
```

Comment:
For future reference, one of the issues with `v22.1.4` was that an error of "`TypeError: environment.setup is not a function`" was being thrown in some repos, there is a discussion [on the Jest GItHub repo](https://github.com/facebook/jest/issues/5119) where it seems like one of the dependencies hasn't yet been updated,